source/tutorials/otu-clustering.rst

```diff
@@ -52,7 +52,7 @@ Dereplicating a ``SampleData[Sequences]`` artifact
 --------------------------------------------------
 
 If you are beginning your analysis with demultiplexed, quality controlled
-sequences, such as those in a `QIIME 1 ``seqs.fna`` file`_ your first step is
+sequences, such as those in a QIIME 1 ``seqs.fna`` file your first step is
```

Comment:
This fixes the unescaped code, but it [doesn't fix the link](https://github.com/turanoo/docs/blame/cb3747713dfe1d2ed188e979b8fe0703c0d5c661/source/tutorials/otu-clustering.rst#L189) that this is supposed to point to.

`QIIME 1 ``seqs.fna`` file` is a link to http://qiime.org/documentation/file_formats.html#post-split-libraries-fasta-file-overview